app/src/test/java/ch/epfl/sdp/PlayerTest.java

```diff
@@ -87,4 +82,19 @@ public void getEntityTypeReturnsUser() {
         Displayable currentPlayer = new Player(0,0,0,"temp", "fake");
         assertEquals(EntityType.USER, currentPlayer.getEntityType());
     }
+
+    @Test
+    public void scoreUpdateWorks() throws InterruptedException {
```

Comment:
This header sounds too generic. It is better practice to give the header something more specific like scoreIncreasesOnDisplacementWithTime. Overall though good job!

Reply:
Yes, you are right, I will change it.